src/Core/src/Primitives/Font.cs

```diff
@@ -23,6 +23,13 @@ public FontWeight Weight
 			private set => _weight = value;
 		}
 
+		bool _disableScaling;
+		public bool EnableScaling
```

Comment:
`EnableScaling` or `EnableAutoScaling`... Not sure.

Reply:
I changed it all to AutoScalingEnabled